### ISSUE TITLE
Add to mod manifest

### ISSUE THREAD

**Issue by epicEaston197** [astralchan/GifImporter#7]
Recently downloaded [resolute ](https://github.com/Gawdl3y/Resolute) and notice it hasn't been added to the [mod manifest ](https://github.com/resonite-modding-group/resonite-mod-manifest) resulting in it being unknown

**Comment by astralchan** [comment]
Currently the mod is broken. I'll have to tinker around to try and fix it.

**Comment by astralchan** [comment]
With the release of version [1.1.9](https://github.com/astralchan/GifImporter/releases/tag/1.1.9), it might be worth looking into this issue again and bumping.

**Comment by astralchan** [comment]
With the release of [1.2.0](https://github.com/astralchan/GifImporter/releases/tag/1.2.0), I have a PR on [issue 122](https://github.com/resonite-modding-group/resonite-mod-manifest/pull/122) on the mod manifest.

**Comment by Krzeszny** [comment]
First of all, it's not an issue with this mod. The mod's fine.

Secondly, this issue is blocked from being resolved. The mod can't be added to the manifest until its AssemblyVersion is fixed, and astralchan isn't able to do it for now, so we'd need a fork of this mod to solve this "issue" (which, again, I'd say is not an issue here, rather an issue to be raised in the manifest).

Because #14 was closed for unknown reasons (likely "won't fix"), this issue should also be closed, as it depends on #14.
